deployment/node-feature-discovery/values.yaml

```diff
@@ -0,0 +1,207 @@
+image:
+  repository: gcr.io/k8s-staging-nfd/node-feature-discovery
+  pullPolicy: IfNotPresent
```

Comment:
If using the staging repo the pull policy should prolly be `Always`

Reply:
++

Reply:
So i would need some education here on which image repos NFD has

in general i think we want this (as part of master) to deploy the latest master image.
what repo should we use ? (staging ok i guess for master builds)

and also when a new NFD version is released which repo should be used ?  (will add as comment)

currently the image tag is set to the chart `AppVersion`.  per other comment this will no longer be the case.
I will add to this section a `tag` attribute so the container image can be properly identified.

Will change the pull policy to Always and add a comment that for released versions this should be set to IfNotPresent.
(as a tagged version is going to be used)

Reply:
> in general i think we want this (as part of master) to deploy the latest master image.
> what repo should we use ? (staging ok i guess for master builds)

Yes, staging is the correct one, as is the case in this PR
 
> and also when a new NFD version is released which repo should be used ? (will add as comment)
 
Release images are served under `k8s.gcr.io/nfd/node-feature-discovery`. Actually it would be good to add this to the `prepare-release.sh` script, too.
 
> currently the image tag is set to the chart `AppVersion`. per other comment this will no longer be the case.
> I will add to this section a `tag` attribute so the container image can be properly identified.
> 
> Will change the pull policy to Always and add a comment that for released versions this should be set to IfNotPresent.
> (as a tagged version is going to be used)

👍  We should add this (changing pull policy) to `prepare-release.sh` as well.